apm-agent-plugins/apm-micrometer-plugin/src/test/java/co/elastic/apm/agent/micrometer/MicrometerMetricsReporterTest.java

```diff
@@ -368,7 +368,6 @@ void testExclusionOfFailedGauge_singleGauge() {
             .isEmpty();
 
         getMetricSets();
-        assertThat(metricsReporter.getFailedMeters().iterator().next().getId().getName()).isEqualTo("gauge1");
```

Comment:
[minor] Was this change required due to upgrading micrometer that have different behavior between 1.0.1 and 1.8.0 or due to the change of instrumented method ?

Reply:
It's a different behavior - in the new version the abstract meters capture exceptions quietly, so we don't get to see that they failed